sphinx/source/docs/user_guide/jupyter.rst

```diff
@@ -71,40 +71,40 @@ Once this is installed, usage is the same as with classic notebooks above.
 Bokeh Server Applications
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-It is also possible to embed full Bokeh server applications that can connect
-plot events and Bokeh's built-in widgets directly to Python callback code.
+You can also embed full Bokeh server applications connecting plot events
+and Bokeh's built-in widgets directly to Python callback code.
 See :ref:`userguide_server` for general information about Bokeh server
-applications, and the following notebook for a complete example of a Bokeh
-application embedded in a Jupyter notebook:
+applications. For a complete example of a Bokeh application embedded in
+a Jupyter notebook, refer to the following notebook:
 
 * :bokeh-tree:`examples/howto/server_embed/notebook_embed.ipynb`
 
 JupyterHub
 ++++++++++
 
-In order to embed Bokeh server applications when running notebooks from your own
-JupyterHub instance, some additional steps are necessary to enable network
-connectivity between the client browser and the Bokeh server running in the
+When running notebooks from your own JupyterHub instance, some additional
+steps are necessary to embed Bokeh server applications and to enable network
+connectivity between the client browser and the Bokeh server running in a
 JupyterLab cell. This is because your browser needs to connect to the port the
-Bokeh server is listening on, but JupyterHub is acting as a reverse proxy
+Bokeh server is listening on. However, JupyterHub is acting as a reverse proxy
 between your browser and your JupyterLab container. Follow all the JupyterLab
-instructions above, then continue with the following steps below.
+instructions above, then continue with the following steps:
 
-First, you must install the ``nbserverproxy`` server extension. This can be done
-by running the command:
+1. Install the ``nbserverproxy`` server extension as follows:
 
-.. code:: sh
+   .. code:: sh
 
     pip install nbserverproxy && jupyter serverextension enable --py nbserverproxy
```

Comment:
@Konntroll have you checked the rendered output for this? Offhand, I would expect the actual code block to need new additional indentation to

Reply:
<img width="765" alt="codeindent" src="https://user-images.githubusercontent.com/17789940/95505042-99d41900-09a5-11eb-897c-2693f8c80f5e.png">

Looks okay to me but let me know if you'd rather adjust it.